templates/kubernetes/overlays/dev/kustomization.yml

```diff
@@ -3,6 +3,8 @@ kind: Kustomization
 
 patchesStrategicMerge:
 - deployment.yml
+<%if eq (index .Params `userAuth`) "yes" %>- auth.yml
+<% end %>
 
 resources:
 - ../../base
```

Comment:
the auth.yml for dev-env is actually not needed, but because half of it is applied from `../../base`, we must provide the missing fields so it doesnt error out